mlrun/api/db/sqldb/models/models_mysql.py

```diff
@@ -532,7 +532,16 @@ class DatastoreProfile(Base, mlrun.utils.db.BaseModel):
         name = Column(String(255, collation=SQLCollationUtil.collation()))
         project = Column(String(255, collation=SQLCollationUtil.collation()))
         type = Column(String(255, collation=SQLCollationUtil.collation()))
-        body = Column(String(1024, collation=SQLCollationUtil.collation()))
+        object = Column("object", JSON)
+
+        @property
+        def full_object(self):
+            if self.object:
+                return self.object
```

Comment:
Usually when implementing this way, we do the JSON serialization (`dumps` and `loads`) in the properties. This means clients just handle python dicts, not caring that they are serialized to JSON. This is a good separation of concerns - if at some point we decide to pickle the objects instead, the client code doesn't need to care.
Unless you have very good reasons to not do it here, please follow the same implementation here.